src/pages/python/index.md

```diff
@@ -10,11 +10,19 @@ To know more about Python, you might want to check <a href='https://www.python.o
 
 ## Python 2 or Python 3
 
-*   The two versions are similar, with knowledge of one switching to writing code for the other is easy.
+*   As a beginner, which should you learn?
+    Answer:Python 2.x
```

Comment:
Is there a reason why you would say to beginners to start learning Python 2 when the general consensus across the board is that beginners should start with Python 3 as Python 2 is nearing support and has less features?